composer.json

```diff
@@ -20,20 +20,22 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "minimum-stability": "dev",
```

Comment:
> as opposed to what will be pulled in on project installs

For the time being php http packages are mostly unstable as well, so you are right, without relying on development versions, not the same packages will be installed. Wonder how we could improve that until stable versions are out.